### PR TITLE
deprecated existing scrap API and replaced it with simpler API

### PR DESCRIPTION
**Pull request by oddbookworm**
Reopening of #3 because my original fork was borked and I had to create a fresh fork. Rather than fighting to get the old PR relinked to the new fork, I'm just reopening this PR

### CODE REVIEW

**Review by MyreMylar — COMMENTED** [review]
All the 2.1.4s now need changing to 2.2.0 s

**Review by MyreMylar — APPROVED** [review]
LGTM still 👍

File: docs/reST/ref/scrap.rst [hunk 1]
```diff
@@ -235,6 +260,57 @@ For an example of how the scrap module works refer to the examples page
...
+   .. deprecated:: 2.2.0
+
    .. ## pygame.scrap.set_mode ##
 
-.. ## pygame.scrap ##
+.. function:: put_text
+
+   | :sl:`Places text into the clipboard.`
+   | :sg:`put_text(text) -> None`
+
```
Comment (robertpfeiffer) [inline docs/reST/ref/scrap.rst:271]: Maybe we should drive the point home that this uses unicode strings, not binary data like the legacy API

File: docs/reST/ref/scrap.rst [hunk 2]
```diff
@@ -235,6 +260,57 @@ For an example of how the scrap module works refer to the examples page
...
+   .. ## pygame.scrap.put_text
+
+.. function:: get_text
+   
+   | :sl:`Gets text from the clipboard.`
+   | :sg:`get_text() -> str`
+
+   Gets text from the clipboard and returns it. If the clipboard is empty,
+   returns an empty string. This is a different clipboard than the rest of 
+   the scrap module, except for ``put_text`` and ``has_text``.
```
Comment (robertpfeiffer) [inline docs/reST/ref/scrap.rst:294]: I'd call "the rest of the module" the "legacy API", and mention that unlike the legacy API, this is guaranteed to work cross platform.

File: src_c/scrap.c [hunk 3]
```diff
@@ -93,6 +100,11 @@ _scrap_init(PyObject *self, PyObject *args)
 {
     VIDEO_INIT_CHECK();
 
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "pygame.scrap.init deprecated since 2.1.4", 1) == -1) {
```
Comment (Starbuck5) [inline src_c/scrap.c:104]: 2.1.4s in this file need to be changed.

File: docs/reST/ref/scrap.rst (hunk 1 above)
Comment (Starbuck5) [reply docs/reST/ref/scrap.rst:271]: I think it's okay like this. People expect a string to mean a "real unicode string" in Python 3.

File: docs/reST/ref/scrap.rst (hunk 2 above)
Comment (Starbuck5) [reply docs/reST/ref/scrap.rst:294]: It will work on the platforms that SDL supports it on, at least.

**Review by Starbuck5 — APPROVED** [review]
Besides that note about the version numbers, this looks great to me. Thanks!

File: src_c/scrap.c (hunk 3 above)
Comment (oddbookworm) [reply src_c/scrap.c]: fixed

File: docs/reST/ref/scrap.rst (hunk 2 above)
Comment (oddbookworm) [reply docs/reST/ref/scrap.rst]: This is also wrong. I have pushed a commit fixing this

**Pull request merged**